web/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "es5",
+  "useTabs": false,
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
```

Comment:
I'd prefer double for consistency with the rest of the codebase, but happy to be outvoted if you feel strongly

Reply:
I have no strong feelings so that's fine by me

Reply:
Double quotes is fine by me 👍